Fix typos in the documentation (#1440)
Kludex/uvicorn: docs/deployment.md

### Running gunicorn worker

> It is also possible to use certificates with uvicorn's worker for gunicorn

Reviewer: ```suggestion
It's also possible to use certificates with uvicorn's worker for gunicorn.
```
